Makefile

```diff
@@ -109,32 +109,18 @@ package:
 build: build-source package
 
 # Perform checks
-checks: check-node-version-parity secret secret-clientid
+checks: check-node-version-parity secret
 
-# Check for secrets.json
+
+# Check for secret and confirm proper clientid for production release
 secret:
-	@if [ "$(CONFIG_ENV)" = "release" ] && [ ! -f $(CALYPSO_DIR)/config/secrets.json ]; \
-	then { \
-		if [ -z "${CIRCLECI}" ]; \
-			then { \
-				echo "$(RED)x calypso$/config$/secrets.json not found. Required file, see docs$/secrets.md$(RESET)"; \
-				exit 1; \
-			} \
-		fi; \
-	} \
-	fi;
-
-
-CLIENT_ID := $(shell node -p "require('$(CALYPSO_DIR)$(/)config$(/)secrets.json').desktop_oauth_client_id")
-
-# Confirm proper clientid for production release
-secret-clientid:
-	@if [ "$(CONFIG_ENV)" = "release" ] && [ ! $(CLIENT_ID) = "43452" ]; \
```

Comment:
Dropping the `[ "$(CONFIG_ENV)" = "release" ]` criterion now causes `make dev-server` to fail with 

```
Makefile:123: *** "desktop_oauth_client_id" must be "43452" in /home/bernie/src/wp-desktop/calypso/config/secrets.json.  Stop.
```

when using and oauth ID different from the official WP.com desktop app, even tho docs still [suggest](https://github.com/Automattic/wp-desktop/blob/develop/docs/secrets.md#calypso-secrets) it's fine to use any generated app secrets.

So I think we either need to add the criterion back, or update our docs.

@adlk

Reply:
(Filed #483 to track this.)